python/usb_console/run_simulation.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/local/bin/python3
+
+from argparse import ArgumentParser
+from state_session import StateSession
+from cmdprompt import StateCmdPrompt
+from data_containers import Logger, Datastore
+from simulation import Simulation
+import json, sys, os, threading, time
+
+class SimulationRun(object):
+    def __init__(self, random_seed, data_dir, device_data):
```

Comment:
What types are these inputs supposed to be?

Reply:
random_seed is an int, data_dir is a string, device_data is a JSON array (see the config files.)

Reply:
hahahahaha python and types....